### PR TITLE
Specify which versions of Xcode are supported, fixes CocoaPods/cocoap…

### PR DESCRIPTION
**Pull request by fulldecent**
Fixes https://github.com/CocoaPods/cocoapods.org/issues/412

### CODE REVIEW

**Comment by dnkoutso** [comment]
While this is an OK change I am not sure we want that as we typically "forget" to update this over time and years.

**Comment by fulldecent** [comment]
Good point, updated to more generic wording

File: README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ CocoaPods manages dependencies for your Xcode projects.
 You specify the dependencies for your project in a simple text file: your `Podfile`. 
 CocoaPods recursively resolves dependencies between libraries, fetches 
 source code for all dependencies, and creates and maintains an Xcode 
-workspace to build your project.
+workspace to build your project. The latest released Xcode versions and the
```
Comment (dnkoutso) [inline README.md]: I would add the word "Typically, ...` atm unfortunately we never guarantee this nor do we test for it.

Comment (fulldecent) [reply README.md]: This word change can be a default commitment. If things are not working as described here then they are a bug which should be reported.

**Pull request merged**